CHANGELOG/CHANGELOG-1.2.md

```diff
@@ -1,3 +1,26 @@
+# v1.2.3 - Changelog since v1.2.1
+
+## Changes by Kind
+
+### Bug or Regression
+
+- Update debian images to buster-v1.9.0 ([#839](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/839), [@mattcary](https://github.com/mattcary))
+
+### Uncategorized
+
+- It is now possible to mount a volume with XFS filesystem and its restored snapshot. ([#838](https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/pull/838), [@leiyiz](https://github.com/leiyiz))
```

Comment:
maybe mention mount at the same time?

Reply:
Discussed offline. Decided not to mess with the auto-generated changelog.